config/locales/views/organizations/en.yml

```
@@ -25,6 +25,40 @@ en:
+        copy_btn: "Copy"
+        link_hint: "Anyone with this link can join with the selected role. The link expires in 7 days."
+      list:
+        heading: "Current members"
+        name: "Name"
+        email: "Email"
+        actions: "Actions"
+        change_role: "Change role"
+        make_role: "Make %{role}"
+        remove: "Remove from organization"
+        remove_confirm: "Remove %{name} from this organization?"
+        actions: "Actions"
```

YAMLlint (1.37.1)

[error] duplication of key "actions" in mapping

(key-duplicates)